### PR TITLE
klient/os: add Environ wrapper

### PR DESCRIPTION
**Pull request by rjeczalik**
Used by os.exec handler to merge environmental variables
for a user process.

### CODE REVIEW

File: go/src/koding/klient/os/environ.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// Environ is a convenience wrapper for environment variables
+// produced by os.Environ.
+type Environ map[string]string
+
+func (e Environ) Encode(m Environ) []string {
```
Comment (ppknap) [inline go/src/koding/klient/os/environ.go]: This function is rather MergeAndEncode. Can we create e.Encode() []string and e.Merge(Environ) Environ ?

Comment (rjeczalik) [inline go/src/koding/klient/os/environ.go]: Sure, will do in a separate PR.

**Review by ppknap — APPROVED** [review]

**Comment by ppknap** [comment]
@cihangir merging this for now. If you have any comments you can put them in depending PRs.

**Comment by cihangir** [comment]
👍
